### ISSUE TITLE
[jat10] deepcognition.io soda bottle competition (10h)

### ISSUE THREAD

**Issue by shadiakiki1986** [teamshadi/ssoc18#2]
Hi @jat10

Let's track our progress on the deepcognition.ai competition here

1. [x] warm up (~2h)
    - [x] play around with deepcognition.ai
    - [x] check out the toilet dataset
2. [x] version 0 of our model (~8h)
    - [x] replicate their toilet dataset model on the soda bottles as is and try to get something that is better than random chance (i.e. with 8 labels, accuracy should be ` > 12.5 %`)
        - achieved accuracy of 14% on validation set (built from 90%/5%/5% split train/validation/test)
    - [x] make 1st submission and check score
        - submission opens on July 29
        - submission scores were 0.4 and 0.5
3. [ ] ~~~version 1 of our model (0h)~~~
    - [ ] ~~~add `VGG` as base layers to benefit from transfer learning. Accuracy here should be better than above 1st model~~~
    - [ ] ~~~submit and check score~~~

**Comment by shadiakiki1986** [comment]
@jat10 and I just faced an "internal server error" on deepcognition.io. We sent a message to support. Let's see what they say

Edit: they replied. It was a memory issue because we're trying small instances. Reducing to batch size = 1 worked.

**Comment by shadiakiki1986** [comment]
@jat10 can you please write down here in this issue your progress so far?

**Comment by jat10** [comment]
I can't give up that easily:

![screen shot 2018-07-04 at 12 35 54 am](https://user-images.githubusercontent.com/6978243/42245781-bad102a4-7f22-11e8-9f58-f60492171994.png)

**Comment by jat10** [comment]
@shadiakiki1986 91% :)
![screen shot 2018-07-04 at 12 46 48 am](https://user-images.githubusercontent.com/6978243/42246050-d697ff96-7f23-11e8-9d94-fc5ea337a928.png)

**Comment by jat10** [comment]
@shadiakiki1986 validation accuracy 99.703% 

![screen shot 2018-07-04 at 2 08 11 am](https://user-images.githubusercontent.com/6978243/42248634-42c26624-7f2f-11e8-9bb5-1818fa55d7ca.png)

**Comment by shadiakiki1986** [comment]
That's fantastic! What was your submission score with this model?
